main.go

```diff
@@ -283,6 +284,16 @@ func main() { // nolint:gocyclo
 		setupLog.Error(err, "unable to create controller", "controller", "UserSignupCleanup")
 		os.Exit(1)
 	}
+	if crtConfig.SpaceConfig().SpaceRequestIsEnabled() {
+		if err = (&spacerequest.Reconciler{
+			Client:         mgr.GetClient(),
+			Namespace:      namespace,
```

Comment:
This looks suspicious. The SpaceRequest controller will need to watch all namespaces in the member clusters. SpaceRequests CRs will be created in the user's namespaces. Not in the member or host operator namespaces as majority of our CRs.

Reply:
+1

Reply:
The host-operator namespace is being passed but not used as of now. It will be used when the logic to watch Space CRs and other resources in host-operator namespace will be implemented.

Reply:
it won't be used much for watching (as this is done by the manager which already has this information - what it should watch) but it will be needed so the controller knows where it should create `Space` CRs for the `SpaceRequests` from the member clusters. 
Let's keep it there.